forge/routes/auth/index.js

```diff
@@ -178,6 +179,9 @@ module.exports = fp(async function (app, opts, done) {
                 reply.setCookie('sid', session.sid, cookieOptions)
                 reply.send({ status: 'okay' })
                 return
+            } else {
+                reply.code(403).send({ error: 'User Suspended' })
```

Comment:
Are we sure 'user suspended' is the only reason `session` may be null?

Reply:
yep, we've already checked that there is a user in the authentication on line 171 above so that when we call `createUserSession()` we know there is a user

```
    createUserSession: async function (app, username) {
        const user = await app.db.models.User.byUsernameOrEmail(username)
        if (user && !user.suspended) {
            return app.db.models.Session.create({
                sid: generateToken(32, 'ffu'),
                expiresAt: Date.now() + DEFAULT_WEB_SESSION_EXPIRY,
                UserId: user.id
            })
        }
        return null
    },
```